db/migrate/20250930015506_remove_unneccessary_indecies.rb

```
@@ -0,0 +1,18 @@
+class RemoveUnneccessaryIndecies < ActiveRecord::Migration[7.1]
```

RuboCop (1.80.2)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)